src/lib/pages/pools/components/pool-details/tables/pool-txs/index.tsx

```diff
@@ -58,12 +58,13 @@ export const PoolRelatedTxsTable = ({
     setCurrentPage(1);
   };
 
-  if (error) return <ErrorFetching dataName="transactions" />;
+  if (isError) return <ErrorFetching dataName="transactions" />;
```

Comment:
Why don't we either move `isError` inside `PoolTxsTable` or move `isLoading` and `empty` outside?